zephyr/Kconfig

```diff
@@ -39,6 +39,11 @@ config BOOT_SIGNATURE_KEY_FILE
 	string "PEM key file"
 	default "root-rsa-2048.pem"
 	help
+	  You can use either absolute or relative path.
+	  In case relative path is used, the build system assumes that it starts
+	  from the directory where the MCUBoot configuration file is located.
```

Comment:
You should probably explain a bit more about the `MCUBoot configuration file` it might not be clear that it is the selected Kconfig configuration file.

Reply:
Ok, I will add information that this relates to KConfig configuration file.